app/services/unwithdraw_service.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class UnwithdrawService
+  def process(edition, user = nil)
+    Document.transaction do
```

Comment:
We'll need to set `requires_new: true` on this as per: https://github.com/alphagov/content-publisher/commit/0bcb0cddf0eba7acbde9aaf8a9cc5b5baac9c072